### PR TITLE
[Cleanup/rubocop] Step2: Indentation and Alignment violations

### PR DESCRIPTION
**Pull request by AliSoftware**
This continues work about #208 and builds on top of previous work done in `toolkit-cleanup` feature branch.

This intermediate/incremental PR focuses on layout violations about Indentation and Alignment only.

### CODE REVIEW

File: ext/drawText/extconf.rb [hunk 1]
```diff
@@ -4,30 +4,35 @@
 
 drawTextDirectory = File.dirname(File.absolute_path(__FILE__))
 compilationDirectory = Dir.pwd
-libDirectory = File.dirname(File.dirname(drawTextDirectory)) + "/lib"
+libDirectory = File.join(File.dirname(File.dirname(drawTextDirectory)), '/lib')
 
 # Copy the makefile required for compilation
-FileUtils.cp(drawTextDirectory + "/makefile.example", compilationDirectory + "/Makefile")
+FileUtils.cp(File.join(drawTextDirectory, 'makefile.example'), File.join(compilationDirectory, 'Makefile'))
```
Comment (AliSoftware) [inline ext/drawText/extconf.rb:10]: Those changes, as well as most of the other ones below in that file, were made manually (and not by `rubocop --auto-correct`) and I grant you that they are unrelated to rubocop violations. But when I saw that rubocop touched that file to fix one of the violations, and that I reviewed the file to ensure the correction seemed ok, I couldn't resist to take the occasion to fix those in the process.

**Review by jkmassel — APPROVED** [review]
This looks nice and straightforward – even if the first commit is _very_ long 😁 

:shipit:

File: ext/drawText/extconf.rb (hunk 1 above)
Comment (mokagio) [reply ext/drawText/extconf.rb:10]: Well done

File: lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb [hunk 2]
```diff
@@ -95,7 +95,7 @@ def self.return_value
 
       def self.authors
         # So no one will ever forget your contribution to fastlane :) You are awesome btw!
-        ["loremattei"]
+        ['loremattei']
```
Comment (mokagio) [inline lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb:98]: Out of scope & Nitpick. Seeing this also made me wonder if we should have a more generic "WordPress|Automattic Mobile Platform Team" user for these? 🤔 

```suggestion
        ['Lorenzo Mattei']
```

**Review by mokagio — APPROVED** [review]
Pragmatically, this doesn't add much to the value of the release toolkit. Still, _I'm so very happy our indentation is tidy and enforced now_ 🤩 😅 

Thank you for getting this done @AliSoftware!

:shipit:

File: lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb (hunk 2 above)
Comment (AliSoftware) [reply lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb:98]: I thought about it too. (also there's a lot of info missing in most of those actions metadata but I plan to address that separately)

Comment (mokagio) [reply lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb:98]: Excellent! 👍
